### PR TITLE
[Release] Verify 7.0.0 RC6 [WIP]

### PR DESCRIPTION
**Pull request by kszucs**

### CODE REVIEW

**Comment by kszucs** [comment]
@github-actions crossbow submit --group verify-rc-source --param release=7.0.0 --param rc=6

**Comment by kszucs** [comment]
@github-actions crossbow submit -g verify-rc-binaries -g verify-rc-jars --param release=7.0.0 --param rc=6

**Comment by kszucs** [comment]
@github-actions crossbow submit verify-rc-source-windows --param release=7.0.0 --param rc=6

**Comment by kszucs** [comment]
@github-actions crossbow submit -g verify-rc-wheels --param release=7.0.0 --param rc=6

**Comment by kszucs** [comment]
@github-actions crossbow submit verify-rc-wheels-windows --param release=7.0.0 --param rc=6

**Comment by kszucs** [comment]
@github-actions crossbow submit verify-rc-wheels-macos-11-arm64 --param release=7.0.0 --param rc=6

**Comment by kszucs** [comment]
@github-actions crossbow submit verify-rc-source-python-macos-arm64 --param release=7.0.0 --param rc=6

File: dev/tasks/linux-packages/apache-arrow/yum/arrow.spec.in [hunk 1]
```diff
@@ -834,6 +834,9 @@ Documentation for Apache Parquet GLib.
 %{_datadir}/gtk-doc/html/parquet-glib/
 
 %changelog
+* Mon Jan 24 2022 Krisztián Szűcs <szucs.krisztian@gmail.com> - 7.0.0-1
```
Comment (kszucs) [inline dev/tasks/linux-packages/apache-arrow/yum/arrow.spec.in:837]: @kou seems like we also need update the linux packaging changelogs in https://github.com/apache/arrow/blob/master/dev/release/post-12-bump-versions.sh

**Comment by kszucs** [comment]
@github-actions crossbow submit --group verify-rc --param release=7.0.0 --param rc=6

Let's see all of the verifications tasks at a single place.

**Comment by kszucs** [comment]
@github-actions crossbow submit --param release=7.0.0 --param rc=6 verify-rc-source-python-macos-arm64 verify-rc-wheels-macos-11-*

File: dev/tasks/linux-packages/apache-arrow/yum/arrow.spec.in (hunk 1 above)
Comment (kou) [reply dev/tasks/linux-packages/apache-arrow/yum/arrow.spec.in:837]: Ah, you're right. I'll do it.

BTW, we also need `post-XX-changelog.sh` that runs `archery release changelog add $version`. See also: ARROW-13460

Comment (kszucs) [reply dev/tasks/linux-packages/apache-arrow/yum/arrow.spec.in:837]: Added a ticket https://issues.apache.org/jira/browse/ARROW-15449

**Comment by kszucs** [comment]
@github-actions crossbow submit --param release=7.0.0 --param rc=6 verify-rc-wheels-macos-11-amd64

**Comment by kszucs** [comment]
@github-actions crossbow submit --param release=7.0.0 --param rc=6 verify-rc-wheels-macos-11-amd64

**Comment by kszucs** [comment]
Closing in favor of https://github.com/apache/arrow/pull/12261

**Pull request closed without merging**